backend/climateconnect_api/utility/notification.py

```diff
@@ -34,7 +36,7 @@ def create_email_notification(receiver, chat, message_content, sender, notificat
         number_of_participants = Participant.objects.filter(chat=chat, is_active=True).count()
         is_group_chat = number_of_participants > 2
         if is_group_chat:
-            if email_settings['email_on_group_chat_message'] == True:
```

Comment:
Maybe we should rather create an env variable disabling notifications all together. 
I think it would make sense to do that in a seperate PR

Reply:
Okay, I will remove this then.